itest/lnd_multi-hop_test.go

```diff
@@ -1805,7 +1805,7 @@ func runMultiHopHtlcAggregation(ht *lntest.HarnessTest,
 		payHash := preimage.Hash()
 		invoiceReq := &invoicesrpc.AddHoldInvoiceRequest{
 			Value:      invoiceAmt,
-			CltvExpiry: thawHeightDelta - 4,
+			CltvExpiry: thawHeightDelta,
```

Comment:
agree my bad for the bad docs, because we are not considering the parent feerate anymore for the closing sweep transaction, all the itest changes are obsolete.